integration_tests/pkg/executor/executor_test.go

```diff
@@ -219,10 +219,11 @@ func testExecutor(t *testing.T, executor Executor) {
 		defer StopAndEraseOutput(taskHandle)
 
 		// Wait for the command to execute.
-		// TODO(bplotka): Remove the Sleep/Wait, since this is prone to errors on different environments.
-		taskHandle.Wait(1 * time.Second)
+		terminated, err := taskHandle.Wait(0)
```

Comment:
Shouldn't we have 60s timeout, so that we won't wait 10minutes if something goes wrong?